src/VerifyWindowsHelloController.js

```diff
@@ -121,7 +124,8 @@ function (Okta, FormController, FormType, webauthn, Spinner, FooterSignout, Wind
       },
 
       postRender: function () {
-        if (this.options.appState.get('factors').length === 1 && !this.model.get('__enrollmentState__')) {
+        if (this.options.appState.get('factors').length === 1 && !this.model.get('__autoTriggered__')) {
```

Comment:
Do we really need the extra prop ```__autoTriggered__```? What I am thinking is that if Windows Hello is the only factor, then we should just open the windows modal. Since it is the only factor, they can't really navigate anywhere else. Or what will be the case where we need to check if ```__autoTriggered__``` is true?